client/.gitignore

```diff
@@ -1,90 +1,19 @@
-# Created by .ignore support plugin (hsz.mobi)
-### Node template
-# Logs
-/logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
-# Runtime data
-pids
-*.pid
-*.seed
-*.pid.lock
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# nyc test coverage
-.nyc_output
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# Bower dependency directory (https://bower.io/)
-bower_components
-
-# node-waf configuration
-.lock-wscript
-
-# Compiled binary addons (https://nodejs.org/api/addons.html)
-build/Release
-
-# Dependency directories
-node_modules/
-jspm_packages/
-
-# TypeScript v1 declaration files
-typings/
-
-# Optional npm cache directory
-.npm
-
-# Optional eslint cache
-.eslintcache
+# See https://help.github.com/ignore-files/ for more about ignoring files.
 
-# Optional REPL history
-.node_repl_history
+# dependencies
+/node_modules
 
-# Output of 'npm pack'
-*.tgz
+# testing
+/coverage
 
-# Yarn Integrity file
-.yarn-integrity
+# production
+/build
+/dist
+/.next
 
-# dotenv environment variables file
-.env
-
-# parcel-bundler cache (https://parceljs.org/)
-.cache
-
-# next.js build output
-.next
-
-# nuxt.js build output
-.nuxt
-
-# Nuxt generate
-dist
-
-# vuepress build output
-.vuepress/dist
-
-# Serverless directories
-.serverless
-
-# IDE / Editor
-.idea
-
-# Service worker
-sw.*
-
-# macOS
+# misc
 .DS_Store
-
-# Vim swap files
-*.swp
+.env
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
We could use Node.js gitignore instead of what react generates, so we avoid any future problems with files being committed the way they shouldn't.  https://github.com/github/gitignore/blob/master/Node.gitignore